### PR TITLE
Switch to subcommands cli API

### LINKED ISSUE #100: Change CLI API
> On a different note, I should probably refactor the csv stuff into a separate cli as you did with this one. `pythonfmu-csvbuilder`, which would allow for customization and separation of concern.

This is a great idea. One comment, should we move to a cli api like `pythonfmu _command_ options` rather than `pythonfmu-_command_ options` closer to package manager api?

_Originally posted by @fcollonval in https://github.com/NTNU-IHB/PythonFMU/pull/95#issuecomment-602455771_

### PR DESCRIPTION
**Pull request by fcollonval**
Fix #100

### CODE REVIEW

File: examples/demo.ipynb [hunk 1]
```diff
@@ -92,7 +92,7 @@
    "source": [
     "## Generate the FMU\n",
     "\n",
-    "The FMU is generate through the command line tool `pythonfmu-builder`."
+    "The FMU is generate through the command line tool `pythonfmu build`."
```
Comment (markaren) [inline examples/demo.ipynb]: generate -> generated while we're at it

File: README.md [hunk 2]
```diff
@@ -51,9 +51,43 @@ optional arguments:
                         If given, canNotUseMemoryManagementFunctions=false
 ```
 
-##### Example: 
+### How do I build an FMU from python code with third-party dependencies?
+
+Often, Python scripts depends on non-builtin libraries like `numpy`, `scipy`,... . _PythonFMU_ does not package a full environment within the FMU. However you can package a `requirements.txt` or `environment.yml` file within your FMU. And using `pythonfmu deploy`, your end user will be able to update is local Python environment. The steps to achieve that:
```
Comment (markaren) [inline README.md]: I'd move this entire paragraph to line 65, replacing the `For the FMU end user` item

File: README.md [hunk 3]
```diff
@@ -51,9 +51,43 @@ optional arguments:
...
+
+1. For the FMU builder user:
```
Comment (markaren) [inline README.md]: And remove the bullet items altogheter

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # PythonFMU
 
-> A lightweight framework that enables the packaging of Python3.x code as co-simulation FMUs.
+> A lightweight framework that enables the packaging of Python 3 code or CSV file as co-simulation FMUs (following FMI version 2.0).
```
Comment (markaren) [inline README.md]: CSV files

**Comment by fcollonval** [comment]
Doen thanks Lars

**Pull request merged**